lms/djangoapps/teams/static/teams/js/spec_helpers/team_spec_helpers.js

```diff
@@ -289,9 +286,6 @@ define([
                 sort_order: 'name'
             },
             {
-                state: {
-                    pageSize: 5
-                },
                 teamEvents: teamEvents,
```

Comment:
no need to pass state as TopicCollection takes the pageSize from the size of results.